source/bindings/asgenstdlib.d

```diff
@@ -0,0 +1,13 @@
+/* This was taken from <https://github.com/dlang/druntime/commit/c59ecbe27b4da4366c7cb8bce4adf0fefe48c9fb>
+ */
+
+////////////////////////////////////////////////////////////////////////// //
+// C API
+
+module bindings.asgenstdilb;
+
+extern(C):
+nothrow:
+@nogc:
+
+char* mkdtemp(char*); // Defined in IEEE 1003.1, 2008 Edition
```

Comment:
Probably not worth its own unit.
You could add it to the Ubuntu backend code where you need it, or maybe to `utils`.
In any case, version it out for compilers which actually do use a recent standard library:

```D
version (GNU) {
    extern(C) char *mkdtemp(char*) nothrow @nogc;
} else {
    import core.sys.posix.stdlib : mkdtemp;
}
```
(haven't tested that, but it should work well with both GDC and LDC)